buildpatches/build_bazel_rules_nodejs.patch

```diff
@@ -0,0 +1,19 @@
+--- a/third_party/github.com/bazelbuild/bazel-skylib/rules/private/copy_common_private.bzl
++++ b/third_party/github.com/bazelbuild/bazel-skylib/rules/private/copy_common_private.bzl
+@@ -64,10 +64,10 @@
+     # Sandboxing for this action is wasteful as well since there is a 1:1 mapping of input
+     # file/directory to output file/directory and no room for non-hermetic inputs to sneak in to the
+     # input.
+-    "no-remote": "1",
+-    "no-remote-cache": "1",
+-    "no-remote-exec": "1",
+-    "no-cache": "1",
+-    "no-sandbox": "1",
+-    "local": "1",
++    # "no-remote": "1",
++    # "no-remote-cache": "1",
++    # "no-remote-exec": "1",
++    # "no-cache": "1",
++    # "no-sandbox": "1",
```

Comment:
Disabling sandboxing, assuming that it doesn't also disable remote execution, could still be useful here.